### ISSUE TITLE
org-trello seems to be inhibitting automatic updating of packages in Marmalade.

### ISSUE THREAD

**Issue by VrRm** [nicferrier/elmarmalade#104]
The current version of `org-trello` in Marmalade is 0.6.3.  But this version depends on `defered-0.3.2`

```org-trello is an available package.

     Status: Available from marmalade -- [Install]
    Archive: marmalade
    Version: 0.6.3
   Requires: request-deferred-0.1.0, deferred-0.3.2, s-1.9.0, dash-2.8.0, emacs-24
    Summary: Minor mode to synchronize org-mode buffer and trello board```


However, that version of deferred isn't present in Marmalade, so trying to install org-trello` fails.  But worse, if you're in `package-list-packages` and try to upgrade all `U, x`, Emacs will generate the error:

```package-compute-transaction: Need package `deferred-0.3.2', but only 0.3.1 is available```

Aborting the upgrade.  Somehow, even though org-trello wasn't an separate package in 24.3 it is considered part of the org upgrade and therefor causes upgrade all to break.  

The work-around is select all for upgrade `U` and then deselect `org-trello` and then upgrade the remainder of the packages `x`.

The best fix for this would probably be to introduce the correct defered package into marmelade.  The an alternate solution would be to rework the org package so that trello is not automatically included in the upgrade.
